licenses/LICENSE-de.flapdoodle.embed.mongo-2.2.0.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
It is not necessary to include this, since it is apache license which is already included.